content/docs/studio/create-view.md

```diff
@@ -42,24 +42,24 @@ You will need to specify advanced settings in the following scenarios.
 
 - **Monorepo:** If you have connected to a
   [monorepo](https://en.wikipedia.org/wiki/Monorepo), then specify the full path
-  to the root directory for the project that you want to connect to.
+  to the root directory of the project that you want to connect.
 
 - **Data remotes:** The metrics and parameters that you want to include in the
-  view may be present in a data remote (cloud storage or another location
-  outside of the Git repo). If you want to include such data in your views, then
-  you will have to grant DVC Studio access to the data remote.
+  view may be present in a <abbr>data remote</abbr> (cloud storage or another
```

Comment:
No such tooltip (yet)

Reply:
Don't we have `remote`? 

Ah, I think this only on my fork. Missed it, sorry. 

![Screen Shot 2021-05-12 at 19 03 08](https://user-images.githubusercontent.com/476310/118007352-be85da00-b354-11eb-9e5d-c2aba43fe8d9.png)